src/issuemd-core.js

```diff
@@ -1,43 +1,43 @@
-'use strict';
+(function (root, factory) {
 
-! function () {
+    'use strict';
```

Comment:
I followed the pattern from [UMD generator](https://github.com/umdjs/umd) to be consistent.